### ISSUE TITLE
Add drop capability

### ISSUE THREAD

**Issue by catamphetamine** [bensmithett/dragster#8]
Hello.
Today I integrated your library into my project, and I also needed to remove 'dragged-over' css class from the container after the dragged element is dropped inside the container.
But if a dragged element is dropped inside the container the 'dragster:leave' event is not fired.
I made it work like this

```
var dragster = new Dragster(element[0]);

element.on("drop", function (event) {

                dragster.dragleave(event);
})
```

It looks like a hack.
Maybe you should support dropping events too.

**Comment by mysteryos** [comment]
Just came across this issue as well. After dropping a file, the hover element stays in place.

Using the above fix for now.

**Comment by juanialt** [comment]
Same problem here!!
I just put a listener inside the "dragster.js" file.

After the "dragleave" listener I put this:

``` javascript
this.el.addEventListener("drop", this.dragleave, false);
```

I think if the element enters but it never leaves the zone is because it's dropped inside it.
I don't know if it's the best solution but it works!!

**Comment by Charizard** [comment]
Stumbled across the same issue. Using @kuchumovn's solution :+1:

**Comment by michaeladamkatz** [comment]
Used solution by halt-hammerzeit. Just want to point out that calling dragster.dragleave(event); in the drop handler has two benefits. One is that it removes the hover graphic (as indicated by others here). The other is it appears that unless you call dragster.dragleave(event); in the drop handler, subsequent drag actions don't trigger the dragster:enter and dragster:leave events correctly.

**Comment by Sylwekqaz** [comment]
I also call reset, because Dragster not firing next `dragster:enter` until I drag inside element new files and drag outside (dragster:leave is thrown) . 

``` javascript
//init
self.Dragster = new Dragster(document);  
...
//ondrop
self.Dragster.reset();
self.dragleave(); //dragster not fire leave event after drop
```

**Comment by zamicol** [comment]
catamphetamine's issue is fixed by #17.
